core/storage/collection/gae_models.py

```diff
@@ -215,8 +215,10 @@ def get_collection_count(cls) -> int:
         """Returns the total number of collections."""
         return cls.get_all().count()
 
-    # TODO(#13523): Change 'model_dict' to domain object/TypedDict to
-    # remove Any from type-annotation below.
+    # TODO(#15911): Here we use type Any because 'convert_to_valid_dict' method
+    # accepts content NDB JSON properties and those NDB JSON properties have
+    # loose typing. So, once we explicitly typed those NDB JSON properties we
```

Comment:
typed --> type

and maybe add a comma after "properties"

Ditto below (3 places in this file, possibly more elsewhere).

Reply:
Done everywhere it is applicable !!